src/cli/commands/config.js

```diff
@@ -83,10 +83,10 @@ export const {run, setFlags, examples} = buildSubCommands('config', {
     }
 
     reporter.info(reporter.lang('configYarn'));
-    reporter.inspect(config.registries.yarn.config);
+    reporter.inspect(JSON.stringify(config.registries.yarn.config, null, 2));
```

Comment:
I think you should use `reporter.log` at this point with the `{force: true}` option.